App.js

```diff
@@ -217,23 +225,20 @@ class App extends React.Component<Props, *> {
                   setTopLevelNavigator(node);
                 }}
                 theme={current === LIGHT_THEME ? 'light' : 'dark'}
+                language={i18n.language}
               />
               {!!SHOW_THEME_TOGGLE &&
-              <TouchableOpacity
-                style={{
-                  padding: 20,
-                  borderWidth: 1,
-                  borderColor: colors.border,
-                  alignItems: 'center',
-                  backgroundColor: colors.card,
-                }}
+              <Button
+                title={`THEME: ${current}`}
                 onPress={() => {
                   const themeToChangeTo = current === LIGHT_THEME ? DARK_THEME : LIGHT_THEME;
                   setAppTheme(themeToChangeTo);
                 }}
-              >
-                <Text style={{ color: colors.text }}>{`THEME: ${current}`}</Text>
-              </TouchableOpacity>}
+              />}
+              {!!SHOW_LANG_TOGGLE && <Button
+                title={`Change lang (current: ${i18n.language})`}
+                onPress={() => changeLanguage(i18n.language === 'fr' ? 'en' : 'fr')}
```

Comment:
is it some kind of leftover from testing?

Reply:
Looks like it could be...

Reply:
Not really, it's left there intentionally. This works just like a "theme" toggle - tool for easier development and it's visibility is managed using env variables. 
It renders language switcher button at the bottom of the screen, so it would be easier to check if translations are working in each single screen by simply toggling between two languages.
'en' - 'fr' pair is picked because French is a second language for us to go :)

Reply:
Apologies - my misunderstanding!